Base/usr/share/man/man1/which.md

```diff
@@ -0,0 +1,25 @@
+## Name
+
+which - show the full path of commands
+
+## Synopsis
+
+```**sh**
+$ which [options] executable
+```
+
+## Description
+
+`which` provides the full path of where an executable is located at the command prompt.
```

Comment:
Still copied from the Linux man page, except you cut some words off so it no longer even makes sense:

> full path of the executables that would have been executed when this argument **_had been entered_ at the shell prompt.**

Delete this and write it in your own words.

Reply:
done